lib/sequent/core/event_store.rb

```diff
@@ -79,6 +79,29 @@ def load_events(aggregate_id)
         [stream.event_stream, events]
       end
 
+      def load_events_for_aggregates(aggregate_ids)
```

Comment:
Simply delegate `load_events` to this method by wrapping `aggregate_id` in an array?